### PR TITLE
Vet clinic database: add "join table" for visits

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
- Understand the different types of relationships between tables. ✔
- Prepare complex queries that answer analytical questions. ✔
- Use primary key & foreign key mechanisms for joining tables. ✔

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -122,4 +122,63 @@ AND A.escape_attempts = 0
...
+FROM visits
+LEFT JOIN animals A ON A.id_animals = visits.id_animal
+ORDER BY COUNT(A.name) DESC
+
+SELECT animals.name, visits.date_visit
+FROM visits
+LEFT JOIN animals ON animals.id_animals = visits.id_animal
+LEFT JOIN vets ON vets.id_vets = visits.id_vet
+ORDER BY visits.date_visit ASC
+LIMIT 1;
```
Comment (SajjadAhmad14) [inline queries.sql:163]: Please add a query to find Maisy Smith's first visit name?

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Elim, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (CarlosE-Feliz) [reply queries.sql:163]: Hey, I appreciate your effort in trying to do your work but I already have that part.

image
 Can you please double-check and leave a comment for the next reviewer so he can just approve my work? thanks in advance!!!

**Comment by CarlosE-Feliz** [comment]
## Note to the next reviewer 😋😋😃😃

As I texted @SajjadAhmad14  and get no answer I'll go ahead and submit the project again.

## This is what he is requesting from me.
image

## I just check my code and saw that I've what he is requested from me.
image

Please double-check my code, in case I'm really missing something I can fix that, and please if I have more than one problem there PLEASE list all of them so I can fix them.

Thanks in advance! Cheers!

**Review by rukundoeric — APPROVED** [review]
Hi @CarlosE-Feliz,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**